test/vegeta/pacers/combined_pacer_test.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pacers
+
+import (
+	"testing"
+	"time"
+
+	vegeta "github.com/tsenart/vegeta/lib"
+)
+
+func TestCombinedPacer(t *testing.T) {
+	pacer1 := vegeta.Rate{Freq: 1, Per: time.Second}
+	pacer2 := vegeta.Rate{Freq: 5, Per: time.Second}
+	pacer, _ := NewCombined([]vegeta.Pacer{pacer1, pacer2}, []time.Duration{5 * time.Second, 5 * time.Second})
```

Comment:
Can we make this more than 2 elements, to make sure the binary search is normally tested with all possible combinations
1. 0
2. > last
3. exact on the value
4. between values.

Reply:
Sure. Done.